### DISCUSSION TITLE
What happenend to the sorting?

### DISCUSSION THREAD

**Discussion by MilesWK** [community/community#146592] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hello! 👋 I for some reason have somehow flipped my feed around, so newest stuff is on bottom. How might I fix this?

**Comment by whiteSHADOW1234** [comment]
looks like it’s been fixed today by reverting to newest-to-oldest chronological sorting.

**Comment by ebndev** [comment]
Hi @MilesWK,

The fix for `feeds sorting in reverse order` was shipped on December 8th. If you are still having issues with this or would like to provide additional details about your experience, please comment directly on [this discussion](https://github.com/orgs/community/discussions/146771#discussion-7653153), and we will be closing this one as resolved.

Thanks for your patience as we were working towards a solution!

**Comment by MilesWK** [comment]
Sorry I forgot to reply back to this. Thanks! This solved my issue!
